client/components/table/table.js

```diff
@@ -217,6 +219,10 @@ Table.propTypes = {
 			 * Boolean, true if this column is the default for sorting. Only one column should have this set.
 			 */
 			defaultSort: PropTypes.bool,
+			/**
+			 * Boolean, true if this column is an identifier for the row.
+			 */
+			isIdentifier: PropTypes.bool,
```

Comment:
Right, in the beginning I wanted to use `rowHeader` but in some tables we have two columns which must be aligned to the left, so that would require to `th`s.

In the other hand, `isIdentifier` could actually be renamed to `isLeftAligned` (or something similar), but adding style properties to JS looks bad to me.

Reply:
Hm, I agree that adding something just for a style isn't great, but if the only thing `isIdentifier` does is style alignment, we're better just being clear about it.

Reply:
Seems fair. That should be fixed now! 441f34b29dec9b2514303f9fb6c68ef8346ecf0b